Package.swift

```diff
@@ -35,7 +35,7 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(name: "PayPalCheckout", url: "https://github.com/paypal/paypalcheckout-ios", .exact("0.112.1"))
+        .package(name: "PayPalCheckout", url: "https://github.com/paypal/paypalcheckout-ios", .exact("1.0.0"))
```

Comment:
We should consider moving PPCP to use a `binaryDependency` instead of an explicit dependency. Right now when CI runs for the `SPMTest` app it runs an old version of NXO since SPMTest points to the main branch of our SDK. Moving to a binary dependency would allow us to pull in the NXO dependency as part of our package graph on version bump PRs vs needing to bump the sample app in a follow up PR.

It would also allow us to obfuscate this module from merchants since right now if a merchant uses SPM they will _always_ see `PayPalCheckout` pulled in as a dependence even if they aren't using the `PayPalNativePayments` module.

Reply:
No - SPMTest is the equivalent to how a merchant would pull in our SDK. Merchants should not pull in `PayPalCheckout` directly, in fact they shouldn't need to know that we need the underlying module at all. SPMTest does not pull it in today it's pulled in via the `dependencies` line in our `Package.swift`, that app does not currently pull `PayPalCheckout` directly.

Reply:
Check out this PR for some history/reference - https://github.com/braintree/braintree_ios/pull/882

Reply:
I created ticket for creating binary dependency instead of explicit dependency for PayPalCheckout. I only saw a few options for epics and I wasn't sure if it was the right one. DTNOR-296